www/src/scss/_overlay.scss

```diff
@@ -62,6 +62,22 @@
   }
 }
 
+.url-container {
+  margin: 16px 8px;
+
+  input[type='text'] {
+    background: #ebebeb;
+    border: 1px solid #ddd;
+    border-radius: 2rem;
```

Comment:
just curious, why `rem` here when the other stuff is `px`?

Reply:
Yeah, sorry just a habit from one of the other projects.